runtime/common/src/lib.rs

```diff
@@ -76,4 +76,14 @@ pub mod parachains {
 			pub const USDT_ASSET_ID: u128 = 1984;
 		}
 	}
+
+	pub mod kusama {
+		pub mod statemine {
```

Comment:
Let's maybe rename both this `statemine` and the `statemint` above to `asset_hub` as that's the new 'official' name for them.

Reply:
done